src/workflow/RunEnergyPlus.cpp

```diff
@@ -240,16 +252,24 @@ void OSWorkflow::runEnergyPlus() {
         if (errContent.find("EnergyPlus Terminated--Fatal Error Detected") != std::string::npos) {
           throw std::runtime_error("EnergyPlus Terminated with a Fatal Error. Check eplusout.err log.");
         }
+
         // TODO: or we use ErrorFile... In which case what's the point of parsing the number of warnings/severe from eplusout.end?
         // Actually, ErrorFile doesn't understand recurring warnings
         // TODO: add a channel filter on the logger to avoid catching all the debug statements in the ErrorFile class
         const openstudio::energyplus::ErrorFile errFile(errPath);
         std::string status = errFile.completedSuccessfully() ? "Completed Successfully" : "Failed";
-        fmt::print("RunEnergyPlus: {} with ", status);
-        fmt::print(fmt::fg(fmt::color::red), "{} Fatal Errors, ", errFile.fatalErrors().size());
-        fmt::print(fmt::fg(fmt::color::orange), "{} Severe Errors, ", errFile.severeErrors().size());
-        fmt::print(fmt::fg(fmt::color::yellow), "{} Warnings.", errFile.warnings().size());
-        fmt::print("\n");
+        if (m_show_stdout) {
+          if (m_style_stdout) {
+            fmt::print("RunEnergyPlus: {} with ", status);
+            fmt::print(fmt::fg(fmt::color::red), "{} Fatal Errors, ", errFile.fatalErrors().size());
+            fmt::print(fmt::fg(fmt::color::orange), "{} Severe Errors, ", errFile.severeErrors().size());
+            fmt::print(fmt::fg(fmt::color::yellow), "{} Warnings.", errFile.warnings().size());
+            fmt::print("\n");
+          } else {
+            fmt::print("RunEnergyPlus: {} with {} Fatal Errors, {} Severe Errors, {} Warnings.\n", status, errFile.fatalErrors().size(),
+                       errFile.severeErrors().size(), errFile.warnings().size());
+          }
+        }
```

Comment:
Unless --show-stdout is passed: do not print the numer of fatal/severe/warnings

And do not style it (colors) unless `--style-stdout` is passed.